.sphinx/_static/footer.css

```diff
@@ -0,0 +1,52 @@
+.display-contributors {
+  color: #ffffff;
+}
+.all-contributors {
+  display: none;
+  z-index: 55;
+  list-style: none;
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  width: 200px;
+  height: 200px;
+  overflow-x: scroll;
+  margin: auto;
+  padding: 0;
+  background: var(--color-background-primary);
+}
+.all-contributors::-webkit-scrollbar {
+  display: none; /* Hide scrollbar for WebKit browsers */
```

Comment:
I was a bit confused that it said "7 contributors" but only displayed 4 - maybe it would be beneficial to have the scrollbar (only to scroll vertically)? Can we style it like the one for the navigation on the right?
![image](https://github.com/canonical/sphinx-docs-starter-pack/assets/11227796/f31217b4-b967-4436-85d3-8b2fd9511039)